demisto_sdk/commands/test_content/test_modeling_rule/test_modeling_rule.py

```diff
@@ -619,6 +709,31 @@ def validate_modeling_rule(
             mr_entity.testdata_path.as_posix()
         )
 
+        if schema_path := mr_entity.schema_path:
+            with open(mr_entity.schema_path) as schema_file:
+                schema = json.load(schema_file)
+        else:
+            raise FileNotFoundError(
+                f"schema file does not exist in path {mr_entity.schema_path}"
+            )
+
+        if (
+            Validations.SCHEMA_TYPES_ALIGNED_WITH_TEST_DATA.value
```

Comment:
If the class inherits from str, you don't need `.value`

```suggestion
            Validations.SCHEMA_TYPES_ALIGNED_WITH_TEST_DATA
```